app/main.ipynb

```diff
@@ -126,19 +161,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[-2, -3]]\n",
+    "\n",
+    "result = np.sum(selected_elements)\n",
+    "\n",
+    "print(\"Selected elements:\", selected_elements)\n",
+    "print(\"Sum of selected elements:\", result)"
```

Comment:
The selected elements for summation do not match the expected result of 27. You need to select the correct subsequence from the array to achieve the desired sum.